lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -346,6 +346,20 @@ interface Account {
         skipValidate: Boolean,
     ): Request<EstimateFeeResponseList>
 
+    /**
+     * @param caller: Authorized executor of the transaction(s):  Hex address or Felt.fromShortString("ANY_CALLER")
+     * @param executeAfter: Unix second timestamp of the beginning of the timeframe
+     * @param executeAfter: Unix second timestamp of the end of the timeframe
+     * @param calls: calls
+     */
```

Comment:
1. [KDoc](https://kotlinlang.org/docs/kotlin-doc.html) doesn't use semicolon (`:`)
2. We're missing `nonce` param 😅 
```suggestion
    /**
     * @param caller Authorized executor of the transaction(s): Hex address or Felt.fromShortString("ANY_CALLER")
     * @param executeAfter Unix second timestamp of the beginning of the timeframe
     * @param executeAfter Unix second timestamp of the end of the timeframe
     * @param calls List of calls to be executed by outside caller
     * @param nonce Nonce for the transaction
     */
```

Reply:
Also, I think the name of method could be simplified, e.g. `signOutsideExecutionCall`
Wdyt @MEJIOMAH17 ?

Reply:
I have committed fixes based on your comments. [Remove : from KDoc](https://github.com/software-mansion/starknet-jvm/pull/552/commits/178875a9330b495a461998e544884d4a2301c498)